### DISCUSSION TITLE
FTP load balancing in Multi-Node-Multi-Drive deployment

### DISCUSSION THREAD

**Discussion by CherrySuryp** [minio/minio#20349] [Q&A] [answered]
Hello everyone👋

We're considering backing up our clients data to MinIO, and it appeared that we'll need FTP protocol to transfer data from servers to storage (due to legacy software on client's side). 

And the question is - how would we balance these uploads between nodes if Nginx doesn't support FTP load balancing?
I'm thinking about using HAProxy since it has option to balance FTP traffic, but is it good option?

**Servers configuration we'll be using:**
MinIO: 1 Pool, 4 Bare Metal nodes, 4x Drives per node. LAN connection between nodes. Linux deployment
Proxy: Single Server with LAN connection to MinIO cluster

Thanks for help in advance!

**Comment by klauspost** [comment, marked as answer]
We (MinIO) don't have any recommendations for a proxy. I am not familiar with existing proxies, so I can only suggest googling.

**Reply by CherrySuryp** [reply]
Okay, thank you for answer. I will write and update here once I'll test FTP load balancing with HAProxy
